includes/platform-checkout-user/class-platform-checkout-save-user.php

```diff
@@ -88,7 +91,10 @@ public function maybe_add_userdata_to_metadata( $metadata, $order ) {
 			$metadata['platform_checkout_secondary_last_name']  = wc_clean( $order->get_shipping_last_name() );
 			$metadata['platform_checkout_secondary_phone']      = wc_clean( $order->get_shipping_phone() );
 			$metadata['platform_checkout_secondary_company']    = wc_clean( $order->get_shipping_company() );
+			$metadata['platform_checkout_source_url']           = $platform_checkout_marketing_source_url;
```

Comment:
I don't see platform_checkout_source_url and platform_checkout_is_blocks being used anywhere. Are we recording them for tracking purposes?

Reply:
It's used on 1568-gh-Automattic/woopay, I included it on this PR so we don't need to open another one just to include these two fields.